web/templates/default_language_modal.hbs

```diff
@@ -1,8 +1,5 @@
 <p>
-    {{#tr}}
-    A language is marked as 100% translated only if every string in the web, desktop,
-    and mobile apps is translated, including administrative UI and error messages.
-    {{/tr}}
+    {{t "A language is marked as 100% translated only if every string in the web, desktop, and mobile apps is translated, including administrative UI and error messages." }}
```

Comment:
I guess we could be line-wrapping this to avoid a super long line if we wanted to?

Reply:
It seems Handlebars string literals are allowed to contain bare newlines (unlike JavaScript string literals), so we could copy the line-wrapping code from `{{#tr}}`.